api/features_api.py

```diff
@@ -375,17 +376,19 @@ def do_patch(self, **kwargs):
 
     return {'message': f'Feature {feature_id} updated.'}
 
-  @permissions.require_create_feature
-  def do_delete(self, **kwargs) -> dict[str, str]:
+  def do_delete(self, **kwargs) -> flask.Response | dict[str, str]:
     """Delete the specified feature."""
     # TODO(jrobbins): implement undelete UI.  For now, use cloud console.
-    feature_id = kwargs.get('feature_id', None)
+    if 'feature_id' not in kwargs:
+      self.abort(404, msg='Feature ID not specified')
+    feature_id: int = int(kwargs.get('feature_id', 0))
```

Comment:
Nit: We might want to also check to confirm here that feature ID is an integer before trying to cast it.

```py
    if not isinstance(kwargs['feature_id'], int):
      self.abort(400, msg='Invalid feature ID.')
```

Not required though, and unlikely to cause much problems. 🙂

Reply:
Good point.  Actually, it should always be an int because it is declared as an int path parameter in main.py.  The cast to int should probably never have been there.